searx/settings.yml

```diff
@@ -1210,6 +1210,36 @@ engines:
   #    query_str: 'SELECT * from my_table WHERE my_column = %(query)s'
   #    shortcut : psql
 
+  - name: presearch
```

Comment:
Could you make it so presearch is disabled by default?

We haven't discussed to add more engines by default.